featuretools/primitives/standard/transform/email/is_free_email_domain.py

```diff
@@ -36,7 +36,7 @@ def get_function(self):
         def is_free_email_domain(emails):
             # if the input is empty return an empty Series
             if len(emails) == 0:
-                return pd.Series([])
+                return pd.Series([], dtype="boolean")
```

Comment:
Why are these changes included?

If this is fixing something, maybe we should also include a note in the release notes?